browser/src/index.tsx

```diff
@@ -294,7 +294,7 @@ const start = (args: string[]) => {
     UI.init()
 
     ipcRenderer.on("menu-item-click", (_evt, message) => {
-        instance.command("normal! " + message)
+        instance.command("exec \":normal! " + message + "\"")
```

Comment:
Hmm, this works from the nvim command line:

    :call nvim_feedkeys("\<c-r>", 'n', v:false)

But of course that "escaped" keycode `"\<c-r>"` is a special thing that won't work outside of vim-land. Using the API from something like python or oni, one would need to send the raw ^R byte (however one's client represents "octal 22"):

    :call nvim_feedkeys("\022", 'n', v:false)

Yuck, that's actually worse than sending the literal string `"norm! "\<c-r>"` to nvim_command().

`nvim_input()` is much more convenient in this regard: `nvim_input('<c-r>')` does what you might expect. But it is raw user input, so one can't disable mappings. 

I should have realized this when I suggested feedkeys(), my mistake.